shell/platform/windows/win32_window.cc

```diff
@@ -108,174 +108,169 @@ void Win32Window::TrackMouseLeaveEvent(HWND hwnd) {
 }
 
 LRESULT
-Win32Window::MessageHandler(HWND hwnd,
-                            UINT const message,
-                            WPARAM const wparam,
-                            LPARAM const lparam) noexcept {
+Win32Window::HandleMessage(UINT const message,
+                           WPARAM const wparam,
+                           LPARAM const lparam) noexcept {
   int xPos = 0, yPos = 0;
   UINT width = 0, height = 0;
-  auto window =
-      reinterpret_cast<Win32Window*>(GetWindowLongPtr(hwnd, GWLP_USERDATA));
   UINT button_pressed = 0;
 
-  if (window != nullptr) {
-    switch (message) {
-      case kWmDpiChangedBeforeParent:
-        current_dpi_ = GetDpiForHWND(window_handle_);
-        window->OnDpiScale(current_dpi_);
-        return 0;
-      case WM_SIZE:
-        width = LOWORD(lparam);
-        height = HIWORD(lparam);
-
-        current_width_ = width;
-        current_height_ = height;
-        window->HandleResize(width, height);
-        break;
-      case WM_FONTCHANGE:
-        window->OnFontChange();
-        break;
-      case WM_MOUSEMOVE:
-        window->TrackMouseLeaveEvent(hwnd);
-
-        xPos = GET_X_LPARAM(lparam);
-        yPos = GET_Y_LPARAM(lparam);
-        window->OnPointerMove(static_cast<double>(xPos),
-                              static_cast<double>(yPos));
-        break;
-      case WM_MOUSELEAVE:;
-        window->OnPointerLeave();
-        // Once the tracked event is received, the TrackMouseEvent function
-        // resets. Set to false to make sure it's called once mouse movement is
-        // detected again.
-        tracking_mouse_leave_ = false;
-        break;
-      case WM_SETCURSOR: {
-        UINT hit_test_result = LOWORD(lparam);
-        if (hit_test_result == HTCLIENT) {
-          window->OnSetCursor();
-          return TRUE;
-        }
-        break;
+  switch (message) {
```

Comment:
Almost everything here is just a whitespace change due to removing the `window != nullptr` check (which was doing nothing; the dispatch to this instance method from the static `WndProc` above was done using exactly the same logic used to get `window` from `hwnd`.